docker-compose.yml

```diff
@@ -3,17 +3,20 @@ version: '3.9'
 services:
 
   localstack:
-    image: localstack/localstack:0.11.1
+    image: localstack/localstack:0.14.3
     ports:
       - "4566:4566"
-      - "4571:4571"
-      - "4578:4578"
     networks:
-      - hocs-network
+      hocs-network:
+        aliases:
+          - localhost.localstack.cloud
     environment:
       HOSTNAME_EXTERNAL: localstack
       DEFAULT_REGION: eu-west-2
-      SERVICES: es, sqs
+      SERVICES: sqs,sns,es,s3
```

Comment:
This doesn't require s3?

Reply:
it seems that es is using s3 for something, I get an exception on es startup if S3 isn't running

Reply:
Its a wrapper around ES so should be nothing using it. 

One to investigate me thinks - I'll give it a look in 5.

Reply:
I've just span this up without the s3. Did it have the healthcheck that used S3 as that would be why. 

Looks like it can be removed.